### PR TITLE
reformatted free memory print() to formatted fstrings

### PR DESCRIPTION
**Pull request by LuciMouse**

### CODE REVIEW

File: examples/plotters_combined.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 from pimoroni_ltr559 import LTR559
 from pimoroni_pms5003 import PMS5003
 
-#print("(", gc.mem_free(), ")")
+print(f"initial memory: {gc.mem_free():,}")
```
Comment (e4r7hbug) [inline examples/plotters_combined.py:39]: Formatting options not necessary.
```suggestion
print(f"initial memory: {gc.mem_free()}")
```

**Review by e4r7hbug — CHANGES_REQUESTED** [review]
Maybe change.

File: examples/plotters_combined.py (hunk 1 above)
Comment (LuciMouse) [reply examples/plotters_combined.py:39]: those are thousand separators for the printed display

Comment (e4r7hbug) [reply examples/plotters_combined.py:39]: Ah, right.

**Pull request merged**